app/api/files/specs/v2_support.spec.ts

```diff
@@ -8,22 +8,45 @@ const factory = getFixturesFactory();
 
 const fixtures: DBFixture = {
   entities: [
-    factory.entity('entity1', 'template1', {
-      relProp: [
-        {
-          value: 'entity2',
-          label: 'entity2',
-        },
-        {
-          value: 'entity3',
-          label: 'entity3',
-        },
-      ],
-    }),
-    factory.entity('entity2', 'template1', {
-      relProp: [{ value: 'entity3', label: 'entity3' }],
-    }),
-    factory.entity('entity3', 'template1'),
+    factory.entity(
```

Comment:
Yes, I don't fully remember at this point. But I had an error in the tests. I can check and circle back.

Reply:
Glad you noticed this. While this changes are needed, this is not the place and I was testing the wrong thing. Long story short: after deleting the text reference, entity2 and 3 are re-denormalized so they must end up with an empty obsoleteMetadata field and an empty relProp field. Why this is needed in this PR? Really likely because now the ignoreMetadata function in the v1-v2 integration sets the value to [] if it's unset, plus the entity save process is now actually triggering a denormalization check. For some reason at some point I fixed the expect by touching the fixtures instead of the expect 😬 Good catch!